COLLADAMaya/scripts/create_version_info_h.bat

```diff
@@ -19,10 +19,7 @@ echo namespace COLLADAMaya >> %F%
 echo { >> %F%
```

Comment:
We don't need that script, right ?
/COLLADAMayaVersionInfo.h is valid as checked in the project